### PR TITLE
waifu2x-caffe: Add version 1.2.0.2

### PR DESCRIPTION
**Pull request by davidxuang**
Adding [waifu2x-caffe](https://github.com/lltcggie/waifu2x-caffe) as requested at #1973.

### CODE REVIEW

**Comment by linsui** [comment]
Please move `bin` and `shortcuts` after `extract_dir` . Remove autoupdate.hash.

**Comment by davidxuang** [comment]
Done.

File: bucket/waifu2x-caffe.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/lltcggie/waifu2x-caffe",
+    "version": "1.2.0.2",
+    "description": "Caffe-based GUI/CLI version of waifu2x, an AI-powered image-upscaling program.",
```
Comment (Ash258) [inline bucket/waifu2x-caffe.json]:
```suggestion
    "description": "AI-powered image upscaler.",
```

**Comment by davidxuang** [comment]
Closed due to branch conflict in the clone repo. Re-pull soon.

**Pull request closed without merging**